tcf_website/templates/base/sidebar.html

```diff
@@ -4,6 +4,15 @@
 <div id="sidebar-wrapper" class="border-right pb-0
      d-flex flex-column justify-content-between text-center">
   <ul class="list-group list-group-flush mt-3">
+    <li>
+        <a href="{% url 'blog' %}"
+            class="list-group-item list-group-item-action pb-0
+        {% if request.resolver_match.url_name == 'blog' %}active{% endif %}">
+        <h4><i class="fa fa-comments fa-fw"></i></h4>
+        <p>NEW! Blog</p>
```

Comment:
For styling the `New` tag, I would check out Bootstrap's badges: https://getbootstrap.com/docs/5.0/components/badge/

See also the way the future [Save Course is styled](https://github.com/thecourseforum/theCourseForum2/blob/1467bd77a3dc7be5e406400fbdb4b5a57e8eddd9/tcf_website/templates/base/sidebar.html#:~:text=%3C!%2D%2D%20temporary%20badge%20%2D%2D%3E,2%22%3ENew%3C/span%3E)
![image](https://user-images.githubusercontent.com/45245651/154771529-9fe73a89-133b-4bf1-b431-76784acd39df.png)